### ISSUE TITLE
Security notice restoration screen - As a user restoring an ID, I need a warning message to ensure I'm entering my code/key into a valid Blockstack app, so I don't get my ID stolen.

### ISSUE THREAD

**Issue by jeffdomke** [blockstack/blockstack-browser#1456]

**Comment by aulneau** [comment]
@jeffdomke can you elaborate more on what you mean by this?

**Comment by markmhx** [comment]
_Bump_ @jeffdomke since I suspect this may still be important but I'm not sure exactly what you mean either.

You mean we need a mechanism that helps prevent fake Browser phishing?

**Comment by jeffdomke** [comment]
Yep, more or less. Not a big problem right now, but might become in the future. Even a simple thing that says "Make sure your browser says `https://browser.blockstack.org`" but even that gets complex on mobile etc. Might be part of a larger simple security measures epic.
